public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -78,7 +80,7 @@ import {
   getAnomalyHistoryWording,
   NUM_CELLS,
 } from '../../AnomalyCharts/utils/anomalyChartUtils';
-import { darkModeEnabled } from '../../../utils/kibanaUtils';
+import { darkModeEnabled } from '../../../utils/opensearchDashboardsUtils';
```

Comment:
You may ignore this change for now - all OpenSearch rebasing changes will come in later PR